src/Analyzers/CSharp/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -108,29 +86,31 @@ static bool IsRemoteApiUsage(SyntaxNode node, SemanticModel model)
         foreach (var diagnostic in diagnostics)
         {
             var node = diagnostic.Location.FindNode(getInnermostNodeForTie: true, cancellationToken);
-            MakeDeclarationNullable(editor, model, node, alreadyHandled, cancellationToken);
+            await MakeDeclarationNullableAsync(
+                document, model, editor, node, alreadyHandled, cancellationToken).ConfigureAwait(false);
```

Comment:
>model [](http://example.com/codeflow?start=26&length=5)

could be lazy so model isn't needed up front here too

Reply:
true.  but i'm ok here deferring that until we have some traces indicating benefit :)